media-video/gaupol/gaupol-0.28.2-r1.ebuild

```diff
@@ -29,7 +29,12 @@ RDEPEND="app-text/iso-codes
 	)"
 DEPEND="${RDEPEND}
 	dev-util/intltool
-	sys-devel/gettext"
+	sys-devel/gettext
+	test? (
+			dev-python/pytest[${PYTHON_USEDEP}]
+			dev-python/pytest-runner[${PYTHON_USEDEP}]
```

Comment:
Just an indentation nitpick: my editor shows 3 tabs here but I think it should be 2 tabs.